wolox-books/src/app/screens/book/components/show/show.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <div>
+    <a class="back" (click)="goBack()">
+      <img src="../../../../../assets/back-arrow.png" alt="back-arrow">
+      <span class="back-text">Atr&aacute;s</span>
+    </a>
+    <div class="card">
+      <img src="../../../../../assets/show-book-cover.png" alt="book-cover" class="book-cover">
+      <img src="../../../../../assets/badge.png" alt="book-cover" class="badge">
```

Comment:
close tag

Reply:
raise an error when compile

Reply:
this `<img src="../../../../../assets/badge.png" alt="book-cover" class="badge" />` show you an error? 😮